src/cdk-experimental/scrolling/virtual-scroll-viewport.scss

```diff
@@ -11,10 +11,17 @@ cdk-virtual-scroll-viewport {
   position: absolute;
   top: 0;
   left: 0;
-  right: 0;
   will-change: contents, transform;
 }
 
+.orientation-horizontal {
```

Comment:
nit: classes must be namespaced, e.g. `.virtual-scroll-orientation-horizontal`

Reply:
Done :)